controllers/util/storageconsumer.go

```diff
@@ -305,3 +308,77 @@ func GetStorageConsumerDefaultResourceNames(
 	}
 	return defaults
 }
+
+// These methods are for backward compatibility for provider mode upgraded cluster
+
+func FillBackwardCompatibleConsumerConfigValues(
```

Comment:
We need a better name here. Also why does this need to move outside of the controllers package?

Reply:
In ocs-op there is a package for each controller and the storageconsumer package has the name "controller". We are using it in the storageCluster package and for storageconsumer hence I needed to move it to a util.